### ISSUE TITLE
Allow changing language of user interface

### ISSUE THREAD

**Issue by yvolk** [andstatus/todoagenda#119]
Currently our app uses Language that is set for the whole Android device ("Device default").
In this change we add "App language" option to Settings -> "Other settings" screen that allows setting any custom language from a list of available translations.

**Comment by ploink** [comment]
Android 13 has included a feature that allows you to change the locale for any app that targets A13 and has the feature enabled. This is very easy to implement.

I submitted the same feature request to Etar and they implemented this.
https://github.com/Etar-Group/Etar-Calendar/issues/1497
I already tried it for Todo Agenda and it works perfectly. Here is a screenshot.
![Screenshot_20231224-175150_Settings](https://github.com/andstatus/todoagenda/assets/3049492/03d9f480-0891-4046-9a9c-f907d3e04bc8)
